agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -19,13 +19,17 @@ package pgstatmonitor
 import (
 	"context"
 	"database/sql"
+	"encoding/json"
 	"fmt"
 	"io"
+	"strconv"
 	"time"
 
+	"github.com/lib/pq"
 	_ "github.com/lib/pq" // register SQL driver.
 	"github.com/percona/pmm/api/agentpb"
 	"github.com/percona/pmm/api/inventorypb"
+	"github.com/percona/pmm/api/qanpb"
```

Comment:
it's better to not import qanpb packages in pmm-agent, right now we can change internal implementation of PMM Server without having any issues with backward compatibility. As I mentioned in `pmm` PR, we can use just a `map` for histograms.

Reply:
@BupycHuk Then marshalling will be done on QAN-API2 side, is this OK?

Reply:
if we will use map, then yes, but if we will use separate message type, then best option for us is to convert from pmm-agent type to qan-api2 type in pmm-managed.

Reply:
Map will sort keys and this is problem here. I will do it by separate message type.